deploy/K8s-onprem-MIG/custom-metrics-server-config.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: adapter-config
+  namespace: monitoring
+data:
+  triton-adapter-config.yaml: |
+    rules:
+    - seriesQuery: 'nv_inference_queue_duration_us{namespace="default",pod!=""}'
```

Comment:
So you are scaling on queue duration. What if there are multiple models?

Reply:
multiple models are not used in this demo, but explained by River in the email

Reply:
This cannot just be a demo. As discussed in the email this must be a general example that shows auto-scaling. If you are just looking for a place to host the exact example used in the blog then this is not the proper location.